packages/node_modules/samples/browser-call-with-screenshare/index.html

```diff
@@ -44,16 +44,46 @@
       <main>
         <h1>Local Screensharing</h1>
         <p>This sample demonstrates sharing your local screen</p>
+
+        <form id="typeCredentials">
+          <fieldset>
+            <legend>Pick Method of Getting Access Token</legend>
+              <div>
+                <input type="radio" id="oauth" name="drone" onchange="handleRadio()" value="oauth" checked>
```

Comment:
Unsure what's this is exactly. I will investigate it. Most likely I think we can remove it.

Reply:
I remembered what happened here. The reason why both inputs are named drone is because it was temporary fix for selecting only one radio in the form. I will investigate it more to see a better implementation.

Reply:
haha, I know that you copied and paste, I actually know exactly where you got it too 😉
I was trying to make a point that as developers we should know exactly what we're putting out. My philosophy is, If you don't understand it, don't add it

Reply:
Thanks @lalli-flores for the advice and I will continue to follow that philosophy! Also, I figured out that works from this article! 

https://www.w3schools.com/tags/att_input_type_radio.asp